kong/clustering/services/sync/rpc.lua

```diff
@@ -194,13 +211,14 @@ local function do_sync()
 
   local ns_delta = ns_deltas.default
   if not ns_delta then
-    return nil, "default namespace does not exist inside params"
+    return error_handler("default namespace does not exist inside params")
   end
 
   local deltas = ns_delta.deltas
+  local is_full_sync = ns_delta.wipe
 
   if not deltas then
-    return nil, "sync get_delta error: deltas is null"
+    return error_handler("sync get_delta error: deltas is null", is_full_sync)
```

Comment:
We need some comments about when and how we should run `do_sync` again.

Reply:
Could you elaborate?

Reply:
Are you commenting at the place you meant to

Reply:
Yes, I don't understand why here we should call do_sync again, what's the difference between other error handling.

Reply:
Same here